packages/discord.js/src/structures/DMChannel.js

```diff
@@ -102,14 +111,22 @@ class DMChannel extends BaseChannel {
   }
 
   // These are here only for documentation purposes - they are implemented by TextBasedChannel
-  /* eslint-disable no-empty-function */
+
+  /* eslint-disable getter-return */
   get lastMessage() {}
+
   get lastPinAt() {}
+
   send() {}
+
   sendTyping() {}
+
   createMessageCollector() {}
+
   awaitMessages() {}
+
   createMessageComponentCollector() {}
+
   awaitMessageComponent() {}
```

Comment:
I've seen other methods like this modified to have `async`. Should these have it too?